docs/tutorials/flow-app-quickstart.mdx

```diff
@@ -68,48 +70,36 @@ config({
   "discovery.wallet": "https://fcl-discovery.onflow.org/testnet/authn" // Mainnet: "https://fcl-discovery.onflow.org/authn"
 })
 ```
+
+The folder structure should look similar to the image below.
+
+![Configuration image](../images/2.png)
+
 📣 **Tip**: It's recommend to replace these values with environment variables for easy deployments across different environments like development/production or Testnet/Mainnet.
```

Comment:
missed typo -> recommended* - also i notice you removed abbreviations elsewhere, might want to remove "It's" as well?